pkg/cmd/pulumi/doc.go

```diff
@@ -322,12 +382,22 @@ func genFunctionDocstring(function *schema.Function, lang string, helper codegen
 	return function.Comment
 }
 
-func renderLiveView(title, docstring string) error {
+func renderLiveView(title, docstring string, r *docstringResolver) error {
 	if ti, _, err := dynamic.LoadTerminfo(os.Getenv("TERM")); err == nil {
 		terminfo.AddTerminfo(ti)
 	}
 	app := tview.NewApplication()
 	reader := newMarkdownReader(title, docstring, styles.Pulumi, app)
+	reader.externalLinkResolver = func(link string, reader *markdownReader) (bool, error) {
+		docstring, err := r.findDocstring(link)
+		if err != nil {
+			return true, err
+		}
+		currentPage, _ := reader.rootPages.GetFrontPage()
+		reader.backstack = append(reader.backstack, location{page: currentPage})
```

Comment:
Might be nice to fetch the span of the currently selected node (if any) and  stick that in the back stack as well